apps/web/src/hooks/useCallWithGasPrice.ts

```diff
@@ -94,7 +93,8 @@ export function useCallWithGasPrice() {
         functionName: methodName,
         args: methodArgs,
         gasPrice,
-        gas: calculateGasMargin(gas),
+        // for some reason gas price is insamely high when using maxuint approval, so commenting out for now
+        // gas: calculateGasMargin(gas),
```

Comment:
if the gas price is not expected probably something wrong with the tx. Let's add the gas back

Reply:
i did add the gas back did this in my other branch. yup it was issue with tx. sure i add back in this branch too